web/src/lib/components/shared-components/change-date.svelte

```diff
@@ -24,10 +24,20 @@
     value: string;
   };
 
-  const timezones: ZoneOption[] = Intl.supportedValuesOf('timeZone').map((zone: string) => ({
-    label: zone + ` (${DateTime.local({ zone }).toFormat('ZZ')})`,
-    value: 'UTC' + DateTime.local({ zone }).toFormat('ZZ'),
-  }));
+  const timezones: ZoneOption[] = Intl.supportedValuesOf('timeZone')
+    .map((zone: string) => ({
+      label: zone + ` (${DateTime.local({ zone }).toFormat('ZZ')})`,
+      value: 'UTC' + DateTime.local({ zone }).toFormat('ZZ'),
+    }))
+    .sort((zoneA, zoneB) => {
+      let numericallyCorrect =
+        Number(zoneA.value.slice(3).replace(':', '.')) - Number(zoneB.value.slice(3).replace(':', '.'));
```

Comment:
`DateTime` also has `offset` property, maybe we can use that instead?

Reply:
Sure! I wasn't aware of this property.
I have changed it to use `offset` property instead now